docs/en/observability/metrics-app-fields.asciidoc

```diff
@@ -348,7 +348,7 @@ The type of the service data is collected from.
 +
 The type can be used to group and correlate logs and metrics from one service type.
 +
-Example: If metrics are collected from Elasticsearch, service.type would be elasticsearch.
+Example: If metrics are collected from {es}, service.type would be `elasticsearch`.
```

Comment:
Interesting. So does substitution not occur within code snippets? That seems like a good exclusion.

Reply:
Yes! It will not error for content in `code` or

```
code blocks
```

which will also be helpful for spelling checks.

Reply:
👍